devtools/riscv64.patch

```diff
@@ -1,59 +1,42 @@
-diff --git PKGBUILD.orig PKGBUILD
-index 1046d6a..0e9938c 100644
---- PKGBUILD.orig
+diff --git PKGBUILD PKGBUILD
+index 8299c8c..c57a06a 100644
+--- PKGBUILD
 +++ PKGBUILD
-@@ -3,9 +3,9 @@
- 
- pkgname=devtools
- pkgver=20210202
--pkgrel=3
-+pkgrel=3.1
- pkgdesc='Tools for Arch Linux package maintainers'
--arch=('any')
```

Comment:
Actually, please don't remove this part. This is intended because the package differs in a qemu-enabled x86 builder (for x86_64) and in the riscv64 repository, so it's not "any".

Reply:
Ok. i will fix it